tests/test_simulator.py

```diff
@@ -40,3 +41,31 @@ def test_empty_scene(sim):
     # test that empty frames can be rendered without a scene mesh
     for _ in range(2):
         obs = sim.step(random.choice(list(hab_cfg.agents[0].action_space.keys())))
+
+
+def _test_keep_agent_tgt():
+    sim_cfg = habitat_sim.SimulatorConfiguration()
+    agent_config = habitat_sim.AgentConfiguration()
+
+    sim_cfg.scene.id = "data/scene_datasets/habitat-test-scenes/van-gogh-room.glb"
+
+    sim = habitat_sim.Simulator(habitat_sim.Configuration(sim_cfg, [agent_config]))
+
+    agent = sim.get_agent(0)
+
+    sim.close()
+    sim = None
+
```

Comment:
Can we add 2 sequential `sim` reinitialization.

Reply:
How about this.